### PR TITLE
Dashboard Pages & Views

### PR DESCRIPTION
**Pull request by RenaudGagne**
Closes #
Closes #

# Changes proposed in this pull request:

- New navigation menu
- More views
- Admin & Mods page
- Category page
- Feature Coming Soon (TM)
- SM & Drugwars Admins & Mods page

Review please

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -22,12 +22,41 @@ const router = new Router( {
       name: 'topic-list',
       component: loadView( 'TopicList' ),
     },
-
+    {
+      path: '/dashboard',
```
Comment (reggaemuffin) [inline src/router.js]: can we prefix all these pages with /admin?

File: src/views/Admin.vue [hunk 2]
```diff
@@ -0,0 +1,195 @@
+<template>
+  <div class="container dashboard">
+    <section class="main-content columns is-fullheight">
+      <aside class="column is-2 is-narrow-mobile is-fullheight section is-hidden-mobile">
+        <h1 class="header">
+          Admins & Mods
+        </h1>
+        <p class="menu-label is-hidden-touch">
+          Navigation
```
Comment (reggaemuffin) [inline src/views/Admin.vue]: The navigation should be its own component to reduce duplication

Comment (RenaudGagne) [reply src/views/Admin.vue]: Thanks @reggaemuffin, will do

File: src/router.js (hunk 1 above)
Comment (RenaudGagne) [reply src/router.js]: do you mean that all of those pages will be under /admin? like:
/admin/dashboard
/admin/categories
...etc ?

or would it be better:
/dashboard/admin
/dashboard/categories
...
?

Comment (reggaemuffin) [reply src/router.js]: sorry, already fixed this :D and yes it is the admin dashboard, so /admin makes sense :)

**Pull request merged**